driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -380,6 +422,7 @@ public String getOperationTypeString() {
      *
      * @return the operationType
      */
+    @Nullable
```

Comment:
1. We need `@Nullable` not only on `getOperationType`, but also on `getOperationTypeString`.
2. Let's document on both methods that they may return `null` only if `$changeStreamSplitLargeEvent` is used.